# Header. Needed for the frame_id
Header header

geometry_msgs/Pose pose
geometry_msgs/Point velocity
geometry_msgs/Vector3 euler_angle